### PR TITLE
Add Support for Description in Attachments

### PR DESCRIPTION
**Pull request by shubhwip**
Issue Link : https://github.com/Javacord/Javacord/issues/1069 
[Message Attachments](https://discord.com/developers/docs/resources/channel#attachment-object) may have a description attribute.

### CODE REVIEW

File: javacord-core/src/main/java/org/javacord/core/entity/AttachmentImpl.java [hunk 1]
```diff
@@ -102,6 +108,11 @@ public String getFileName() {
         return fileName;
     }
 
+    @Override
+    public String getDescription() {
+        return description;
```
Comment (montlikadani) [inline javacord-core/src/main/java/org/javacord/core/entity/AttachmentImpl.java:113]: Make this method as `Optional` as the `description` might return `null`.

Comment (shubhwip) [reply javacord-core/src/main/java/org/javacord/core/entity/AttachmentImpl.java]: Nice suggestion !!

**Review by KILLEliteMaste — APPROVED** [review]
Thank you for implementing the Message Attachment Description! :) LGTM!
Just a small note: Next time please squash your commits into a single commit. This helps us to have maintain a cleaner commit history

**Comment by KILLEliteMaste** [comment]
@all-contributors please add @shubhwip for code

**Pull request merged**